examples/secured-client-with-basic-auth/secured_client_with_basic_auth.out

```diff
@@ -0,0 +1,5 @@
+# Create a file named `secured_client_with_basic_auth.bal`, copy the code
+# into it, and use the `ballerina run` command to start the service.
+$ ballerina run secured_client_with_basic_auth.bal
+ballerina: started HTTPS/WSS endpoint 0.0.0.0:9090
+2018-07-31 18:00:07,931 INFO  [] - Hello, World!!!
```

Comment:
Shall we remove the date here? 
"INFO  [] - Hello, World!!!" will be enough.

Reply:
Fixed.